chsdi/models/vector/bafu.py

```diff
@@ -2161,3 +2161,25 @@ class Hochwasserstatistik(Base, Vector):
     the_geom = Column(Geometry2D)
 
 register('ch.bafu.hydrologie-hochwasserstatistik', Hochwasserstatistik)
+
+
+class HydrogeologieMarkierversuche(Base, Vector):
+    __tablename__ = 'hydrogeologie_markierversuche'
+    __table_args__ = ({'schema': 'hydrogeolog', 'autoload': False})
+    __bodId__ = 'ch.bafu.hydrogeologie-markierversuche'
+    __template__ = 'templates/htmlpopup/hydrogeologie_markierversuche.mako'
+    __queryable_attributes__ = ['ort', 'milieu', 'markierstoff']
+    __label__ = 'id'
+    id = Column('id', Integer, primary_key=True)
+    nr_meldeblatt = Column('nr_meldeblatt', Integer)
+    y = Column('y', Float)
+    x = Column('x', Float)
+    ort = Column('ort', Unicode)
+    datum = Column('datum', Date)
```

Comment:
You cannot know that obviously but, there is a custom class that parses dates automatically.

```
from chsdi.models.typs import DateTimeChsdi
```

It will probably look nicer in the mako ;)